Detectors/GlobalTrackingWorkflow/src/TOFMatcherSpec.cxx

```diff
@@ -104,6 +104,16 @@ void TOFMatcherSpec::run(ProcessingContext& pc)
   recoData.collectData(pc, *mDataRequest.get());
   const auto clustersRO = pc.inputs().get<gsl::span<o2::tof::Cluster>>("tofcluster");
 
+  LOG(INFO) << "isTrackSourceLoaded: TPC -> " << recoData.isTrackSourceLoaded(o2::dataformats::GlobalTrackID::Source::TPC);
+  LOG(INFO) << "isTrackSourceLoaded: ITSTPC -> " << recoData.isTrackSourceLoaded(o2::dataformats::GlobalTrackID::Source::ITSTPC);
+  LOG(INFO) << "isTrackSourceLoaded: TPCTRD -> " << recoData.isTrackSourceLoaded(o2::dataformats::GlobalTrackID::Source::TPCTRD);
+  LOG(INFO) << "isTrackSourceLoaded: ITSTPCTRD -> " << recoData.isTrackSourceLoaded(o2::dataformats::GlobalTrackID::Source::ITSTPCTRD);
+
+  bool isTPCused = recoData.isTrackSourceLoaded(o2::dataformats::GlobalTrackID::Source::TPC);
+  bool isITSTPCused = recoData.isTrackSourceLoaded(o2::dataformats::GlobalTrackID::Source::ITSTPC);
+  bool isTPCTRDused = recoData.isTrackSourceLoaded(o2::dataformats::GlobalTrackID::Source::TPCTRD);
+  bool isITSTPCTRDused = recoData.isTrackSourceLoaded(o2::dataformats::GlobalTrackID::Source::ITSTPCTRD);
```

Comment:
Also, with the RecoContainer you will have the same uniform interface for input types as other matches, e.g.:
https://github.com/AliceO2Group/AliceO2/blob/bd7be7b02e6ccc8c5545e64021a9af46cdb7bf35/Detectors/GlobalTrackingWorkflow/src/tpcits-match-workflow.cxx#L36

Reply:
About these last three comments.
I expected such comments anf this is the reason I didn't yet implemented the new writer and some features.
I am available to discuss how to move to RecoContainer directly in the MatchTOF class